src/metaschema/oscal_profile_metaschema.xml

```diff
@@ -171,7 +171,7 @@
       <description>Set parameters or amend controls in resolution</description>
       <model>
          <assembly ref="set" max-occurs="unbounded">
```

Comment:
It might be more consistent to use `parameter-setting` as the assembly name instead of `set`.